### ISSUE TITLE
[BUG] Feature checking not working on Chromium < 114

### ISSUE THREAD

**Issue by cwoolum** [oddbird/popover-polyfill#208]
**Describe the bug**
When using older versions of chromium that didn't support popover, the polyfill incorrectly assumes that popover is natively supported and the functionality is broken.

**To Reproduce**

1. Download one of the old versions of chromium before. The official first support release of the popover api was Chrome 114. 
  * [Chrome 114](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Win_x64%2F1135619%2Fchrome-win.zip?alt=media) - Does work as expected
  * [Chrome 113](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Win_x64%2F1121477%2Fchrome-win.zip?alt=media) - Demo site does say popover is supported but it doesn't work
  * [Chrome 110](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Win_x64%2F1084030%2Fchrome-win.zip?alt=media) - Demo site does say popover is supported but it doesn't work
  * [Chrome 109](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Win_x64%2F1070094%2Fchrome-win.zip?alt=media) - Demo site does say popover is supported but it doesn't work
  * [Chrome 108](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Win_x64%2F1058968%2Fchrome-win.zip?alt=media) - Demo site says popover is not supported and the polyfill is applied correctly.
  * [Chrome 104](https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Win_x64%2F1012736%2Fchrome-win.zip?alt=media) - Demo site says popover is not supported and the polyfill is applied correctly.
2. Open the browser and load the popover polyfill test site - https://popover-polyfill.netlify.app/

I used Windows in all cases.

**Expected behavior**
I would expect the polyfill to work in versions 109-113 of chromium 

**Screenshots**
Here is a screenshot of the issue on Chrome 110.
![Screenshot 2024-05-28 212815](https://github.com/oddbird/popover-polyfill/assets/942415/268b2d7c-02c8-4273-9600-c5d8501970f9)

**Browsers tested on**
Windows, Chrome 104-114, Desktop X64

**Additional context**
We are trying to use Chrome 113 for backwards compatibility testing of our [library](https://github.com/qwikifiers/qwik-ui). We could go back to an older version of chromium but the newest version that doesn't support popover would be ideal.

**Comment by jamesnw** [comment]
@cwoolum Interesting- I'm not able to replicate in Chrome 110. This is what I'm seeing-

![20240529_19_40_31_chrome_110](https://github.com/oddbird/popover-polyfill/assets/167908/ebc6cfe9-4928-4465-bb75-f1b4f13297ac)
![20240529_19_48_13_chrome_110](https://github.com/oddbird/popover-polyfill/assets/167908/c9be7fe5-cb42-4a83-8bb9-339d589ae0bc)

The [`isSupported`](https://github.com/oddbird/popover-polyfill/blob/main/src/popover.ts#L15-L21) logic isn't complex, and hopefully we can get more info there.

What is the result of `'popover' in HTMLElement.prototype` in your Dev Tools Console in Chrome 110, ideally on chrome://version/?

Do you have any browser extensions that could be impacting this? Any other ideas on how to replicate?

**Comment by cwoolum** [comment]
I just realized I used Chrome all over the place when I meant Chromium.

These are all clean unzips(not even installed) from the artifact repos so there are no profiles or extensions. Checking if `popover` exists on prototype does return true for me.

![image](https://github.com/oddbird/popover-polyfill/assets/942415/f6a8f4f7-0519-42a0-a10f-66b8efe4daac)

**Comment by jamesnw** [comment]
Can you confirm that you have the Experimental Web Platform Features feature flag disabled? In Chromium 110, enabling that enables popover. If that's disabled, do you have any other feature flags enabled? 

Also, it looks like you have a long list of Active Variations- those often are related to feature flags, or if you are launching Chromium from the command line, controlled with `--enable-features` and `--disable-features`. Are any of those enabling Popover?

**Comment by cwoolum** [comment]
I'm not launching Chromium with any special features. For all of this testing, I'm just clicking on `Chrome.exe` in the unzipped folder. I didn't find any flags related to popover specifically.

It looks like experimental web platform features are disbled.
![image](https://github.com/oddbird/popover-polyfill/assets/942415/88bbbf97-9e90-4506-b402-caf14fe14908)

Are you not able to replicate the issue by using one of the zips above?

**Comment by jamesnw** [comment]
I'm on Mac, so I wasn't able to confirm with those downloads. I was able to find an old Mac version from the Chromium snapshots and replicate what you're seeing.

What I'm guessing is that Chromium version 110 had partial experimental support for popover, hidden behind a feature flag, but different enough from the final spec that popover written to the current spec do not work. For some reason, the Chromium snapshots that you're using have this enabled, (and not through a feature flag you can disable).  

In other words, you're testing what a user would see if they had Chromium version 110 with the experimental popover enabled (which is possible, but unlikely). My recommendation would be for you to find a way to disable that variation (or find a Chromium download without that enabled), so you can test what a more likely user would see. 

I wasn't able to find a way to disable it, but if you are able to, please report back!

**Comment by cwoolum** [comment]
I wasn't able to find a way to disable the flag either. It looks like 108 is pretty similar in terms of other functionality so I'll probably just go with that. It might be good to just have a note staying that Chromium 109-113 are broken although, like you said, it's very unlikely someone would be using those versions.

**Comment by jamesnw** [comment]
I've opened a PR to add a general caveat to the readme- thanks for reporting back!

**Comment by mfreed7** [comment]
Just to quickly chime in here, this is a bit odd. I'm not sure what's going on, since you're correct that the feature was [enabled by default in M114](https://chromiumdash.appspot.com/commits?revision=r1121894&platform=Mac), and before that the `popover` attribute should not have been included without the feature being enabled. I did do a bit of experimentation via Finch on pre-stable releases, to make sure there weren't compat problems with the `popover` attribute name, as there were with `popup`. But I don't think those should have affected the Chromium downloads linked above.

I bisected the addition of `popover` to the HTMLElement prototype, and it happened [somewhere in this range](https://chromium.googlesource.com/chromium/src/+log/15bf9bf5d37af1b07784885cd291002849af01c3..266ae1129d26fa9cd72966e10f905678ce90c988). I definitely made a bunch of popover related changes in that range, but none of them that I can see should have done that! Let me know if you see something there that I missed.

TL;DR: sorry. Hopefully it works to test with 108?

**Comment by jamesnw** [comment]
@mfreed7 Thanks for taking a look. As best as I can understand at this point-

- The Chromium builds **do** have the feature enabled. `showPopover()` works without the feature flag turned on
- I suspect it is one of the "Active Variations" shown on the `chrome:versions` page, but don't know how to match the hash to a feature, or how to control which ones are enabled
- Chrome releases before v114 (as tested on SauceLabs) do not have `popover` on the prototype

So it isn't that `'popover' in HTMLElement.prototype` is true but popover isn't enabled, the issue is that popover is enabled (so the polyfill can't be applied), but it doesn't work according to the latest spec.

**Comment by mfreed7** [comment]
> So it isn't that `'popover' in HTMLElement.prototype` is true but popover isn't enabled, the issue is that popover is enabled (so the polyfill can't be applied), but it doesn't work according to the latest spec.

Right thanks, I understood that. I'm just not sure how it's getting enabled in pre-114 versions. If you want to attach the variations from e.g. 112, I can look them up and see if there's something popover-related there.

**Comment by jamesnw** [comment]
Sure, here's a list of the Active variations. I assume the list of variations that were active from 109 to 113 will be more useful, but it just narrows the haystack down to 100 variations.

<details><summary>Variations that are active in 109, 111, and 113</summary>
1584cf60-3f4a17df
15d3083e-5ce60213
160d8d8d-9d12ca0c
17b84626-3f4a17df
18324944-3f4a17df
186d6e2c-36c0e608
1acce950-3f4a17df
1bb6a450-3f4a17df
1d0518a-3f4a17df
1fce7d57-3f4a17df
206d80d-3f4a17df
250dda8b-3f4a17df
255dfea8-cf12f279
263848e4-3f4a17df
26db9e79-f430c221
2da2abac-b7f59038
2f7e7ede-3f4a17df
3042ad4b-ad2fa222
31af02a2-3f4a17df
363012bc-3f4a17df
36d5ee52-3f4a17df
371f259c-3f4a17df
3b96a1d-3f4a17df
3e8fe123-3f4a17df
42f0e0ea-75d6947c
42f1f10d-98837767
4749874c-455e925b
4852ec7f-3f4a17df
4949c1d8-3f4a17df
4d936449-fd549ada
4ea303a6-3f4a17df
4ff8f5b5-caf7a452
520b2a89-88bf9f37
5349039a-3f4a17df
55ba4cfa-3f4a17df
57675af7-3f4a17df
595f5eb0-f23d1dea
5c4d440e-58c8ac88
5f36436a-f799c15e
625646ce-3f4a17df
65570806-377be55a
68f0d29d-3f4a17df
6becb1e-a6ea97a2
7760b5b2-3f4a17df
7c2504d0-3f4a17df
7f6bf080-3f4a17df
7fb629a1-60fdb59
81b1a2c3-3f4a17df
870c1db8-3f4a17df
87f33ad6-3f4a17df
8963b549-3f4a17df
8bccc03b-3f4a17df
8d7344de-3f4a17df
911e33b9-3f4a17df
931c5f72-3f4a17df
95bc6922-3f4a17df
9909b8ac-3f4a17df
9a564e2a-3f4a17df
9e5c75f1-30e1b12b
9f476f76-3f4a17df
9fe21c85-3f4a17df
a2fd384c-cc71bb94
a779bb20-3f4a17df
a79ba57a-f23d1dea
aa21b4f1-3f4a17df
ad4acdda-3f4a17df
ade3efeb-e1cc0f14
afb5d7b8-3f4a17df
b0f15b33-b0f15b33
b1ceb06f-3f4a17df
baee3c29-3f4a17df
bde7927d-bab20a64
bef5c006-3f4a17df
c297985a-3f4a17df
ca5a2953-ff983c32
caa76e48-caa76e48
cbc04857-3f4a17df
d3566fbd-c6f74b94
d6284ba0-3be74c4d
d664a1aa-3f4a17df
d92d97b4-e913bec6
d990c4ac-3f4a17df
da493d3c-3f4a17df
db59f83a-3f4a17df
df319cb2-27e0e1e8
e11c0dd9-3586c521
e521d2ef-3f4a17df
e52d4c86-3f4a17df
e5726113-3f4a17df
e898a92c-3f4a17df
e8c68789-49a20295
ed2101b4-3f4a17df
eddd0d82-3f4a17df
ef4764d7-c9f4d4ef
f2855e3d-de2b6078
f3ed486d-3f4a17df
f654ad46-c94f66c2
f6e27768-f695fd79
f8b7087b-3d47f4f4
fd051c38-3f4a17df
</details>

<details><summary>Active Variations, Chromium 109.0.5414.0 (Developer Build) (arm64) </summary>
313957be-3ef44cd2
df319cb2-27e0e1e8
8963b549-3f4a17df
3e7d7783-f38a9353
586e7649-cc1e01dd
e5726113-3f4a17df
3e8fe123-3f4a17df
ba227809-3f4a17df
c5480a51-3f4a17df
aa21b4f1-3f4a17df
d6284ba0-3be74c4d
9d165456-3f4a17df
13e2821-46a4c4e1
eddd0d82-3f4a17df
53879ff4-3f4a17df
f406744f-3f4a17df
f2855e3d-de2b6078
870c1db8-3f4a17df
42f0e0ea-75d6947c
87f33ad6-3f4a17df
afb5d7b8-3f4a17df
4d936449-fd549ada
f0f05f6-3f4a17df
88e143f7-3874ee8f
280b9fbc-3f4a17df
1bb6a450-3f4a17df
1a80e9be-3898461f
250dda8b-3f4a17df
ef1cfe77-3f4a17df
9f476f76-3f4a17df
78634d5f-2ba5dc1c
4749874c-455e925b
7c2504d0-3f4a17df
c98a686c-3f4a17df
251b2579-3f4a17df
1acce950-3f4a17df
dbf7a8af-3f4a17df
206d80d-3f4a17df
a2fd384c-cc71bb94
e521d2ef-3f4a17df
55ba4cfa-3f4a17df
f654ad46-c94f66c2
f8b7087b-3d47f4f4
5c4d440e-58c8ac88
fd051c38-3f4a17df
cbc04857-3f4a17df
65570806-377be55a
d0143dc1-3f4a17df
42f1f10d-98837767
363012bc-3f4a17df
18324944-3f4a17df
520b2a89-88bf9f37
1584cf60-3f4a17df
81b1a2c3-3f4a17df
d92d97b4-e913bec6
4852ec7f-3f4a17df
483d5a5e-d379aaa3
263848e4-3f4a17df
3042ad4b-ad2fa222
3fd33f16-da51fc56
c297985a-3f4a17df
90bfc781-3f4a17df
8b5e9272-5124c949
80a52ca2-9c6dd96d
62cb9b8b-41986bdd
f588ef31-3f4a17df
c7053c9f-9a93b053
911e33b9-3f4a17df
829fa6ed-3f4a17df
95bc6922-3f4a17df
36d5ee52-3f4a17df
332a4d9b-b3a17800
26db9e79-f430c221
2f7e7ede-3f4a17df
e79de56c-dee0823
357a64de-dee0823
8bccc03b-3f4a17df
7d6e65a8-89a4b7af
eabb590b-3f4a17df
19ca6892-3f4a17df
5349039a-3f4a17df
d664a1aa-3f4a17df
625646ce-3f4a17df
2fb38473-3f4a17df
fcad39d1-3f4a17df
491f1531-3f4a17df
a79ba57a-f23d1dea
9e5c75f1-30e1b12b
15ee6537-3f4a17df
c4e32a07-cd09e241
255dfea8-cf12f279
ed2101b4-3f4a17df
ca5a2953-ff983c32
da493d3c-3f4a17df
f3ed486d-3f4a17df
411ef982-7c857e
b3c54bb3-a04d2988
bde7927d-bab20a64
e711bfd-3f4a17df
7989c213-3f4a17df
3482a891-410c5d63
4ff8f5b5-caf7a452
234de0a0-ace4e138
b7a22696-2cd5fcd8
3c68a36c-3f4a17df
7b9d3e0c-1dfe4688
d3566fbd-c6f74b94
a779bb20-3f4a17df
ea23a088-5c589c16
e898a92c-3f4a17df
5f36436a-f799c15e
7fb629a1-60fdb59
2da2abac-b7f59038
186d6e2c-36c0e608
4ea303a6-3f4a17df
4949c1d8-3f4a17df
fc7e4d22-3f4a17df
baee3c29-3f4a17df
9fe21c85-3f4a17df
68f0d29d-3f4a17df
15d3083e-5ce60213
bdb15e19-3f4a17df
c9e4cf65-802823a4
9a564e2a-3f4a17df
31af02a2-3f4a17df
371f259c-3f4a17df
d990c4ac-3f4a17df
a18444ea-a18444ea
b0ca2a47-3f4a17df
ef4764d7-c9f4d4ef
9909b8ac-3f4a17df
e52d4c86-3f4a17df
baacd6e1-901f2837
1fce7d57-3f4a17df
7760b5b2-3f4a17df
e8c68789-49a20295
caa76e48-caa76e48
b0f15b33-b0f15b33
94f1fa38-ca72ea58
2b0207ee-2b0207ee
34a9ddc3-54a64e37
ad4acdda-3f4a17df
931c5f72-3f4a17df
ade3efeb-e1cc0f14
b1ceb06f-3f4a17df
5e7b62b9-3f4a17df
ae82076d-3f4a17df
db59f83a-3f4a17df
57675af7-3f4a17df
160d8d8d-9d12ca0c
1d0518a-3f4a17df
7f6bf080-3f4a17df
6becb1e-a6ea97a2
f6e27768-f695fd79
3b96a1d-3f4a17df
dba92675-f23d1dea
595f5eb0-f23d1dea
38ac144c-3f4a17df
bef5c006-3f4a17df
17b84626-3f4a17df
8d7344de-3f4a17df
2856aa31-3f4a17df
e11c0dd9-3586c521
</details>

<details><summary>Active Variations, Chromium 111.0.5533.0 (Developer Build) (arm64) </summary>
```
2510663e-73703436
df319cb2-27e0e1e8
8963b549-3f4a17df
3e7d7783-a4655118
e5726113-3f4a17df
3e8fe123-3f4a17df
c5480a51-3f4a17df
15e85316-3f4a17df
aa21b4f1-3f4a17df
2e42ee4a-ade68764
d6284ba0-3be74c4d
13e2821-c0236c9e
eddd0d82-3f4a17df
13427e22-3f4a17df
f2855e3d-de2b6078
8c153587-3f4a17df
870c1db8-3f4a17df
89a16501-3f4a17df
1d167f26-3f4a17df
42f0e0ea-75d6947c
87f33ad6-3f4a17df
afb5d7b8-3f4a17df
4d936449-fd549ada
88e143f7-e3ef03f6
1bb6a450-3f4a17df
250dda8b-3f4a17df
b6c668dd-3f4a17df
ef1cfe77-3f4a17df
9f476f76-3f4a17df
4749874c-455e925b
7c2504d0-3f4a17df
c98a686c-3f4a17df
1acce950-3f4a17df
206d80d-3f4a17df
a2fd384c-cc71bb94
e521d2ef-3f4a17df
55ba4cfa-3f4a17df
f654ad46-c94f66c2
f8b7087b-3d47f4f4
5c4d440e-58c8ac88
fd051c38-3f4a17df
cbc04857-3f4a17df
65570806-377be55a
d0143dc1-3f4a17df
42f1f10d-98837767
363012bc-3f4a17df
92d2eb18-fa10226e
18324944-3f4a17df
520b2a89-88bf9f37
1584cf60-3f4a17df
81b1a2c3-3f4a17df
dfa02032-7ae17237
565dffc5-565dffc5
d92d97b4-e913bec6
4852ec7f-3f4a17df
483d5a5e-d379aaa3
263848e4-3f4a17df
3042ad4b-ad2fa222
3fd33f16-da51fc56
c297985a-3f4a17df
eac1a994-3f4a17df
90bfc781-3f4a17df
e0e63e5f-decb98bf
f588ef31-3f4a17df
911e33b9-3f4a17df
829fa6ed-3f4a17df
95bc6922-3f4a17df
36d5ee52-3f4a17df
7a842d4a-b3a17800
26db9e79-f430c221
17cd3426-3f4a17df
2f7e7ede-3f4a17df
e79de56c-dee0823
357a64de-dee0823
8bccc03b-3f4a17df
5349039a-3f4a17df
d664a1aa-3f4a17df
625646ce-3f4a17df
53c131d9-e3ef03f6
a79ba57a-f23d1dea
9e5c75f1-30e1b12b
c4e32a07-cd09e241
255dfea8-cf12f279
ed2101b4-3f4a17df
1527f29f-3f4a17df
b3c9749a-3f4a17df
25fa4483-b9406639
ca5a2953-ff983c32
da493d3c-3f4a17df
f3ed486d-3f4a17df
5c783e42-8f8fa88f
b3c54bb3-a04d2988
bde7927d-bab20a64
3482a891-410c5d63
4ff8f5b5-caf7a452
234de0a0-ace4e138
68971777-6f116e34
3c68a36c-3f4a17df
ca12356a-3f4a17df
7b9d3e0c-1dfe4688
6f11e8a1-3f4a17df
d3566fbd-c6f74b94
a779bb20-3f4a17df
ea23a088-5c589c16
e898a92c-3f4a17df
5f36436a-f799c15e
7fb629a1-60fdb59
2da2abac-b7f59038
186d6e2c-36c0e608
4ea303a6-3f4a17df
4949c1d8-3f4a17df
e3fd1192-3f4a17df
de95f00f-3f4a17df
fc7e4d22-3f4a17df
baee3c29-3f4a17df
9fe21c85-3f4a17df
68f0d29d-3f4a17df
15d3083e-5ce60213
d5a65352-3f4a17df
c9e4cf65-802823a4
9a564e2a-3f4a17df
31af02a2-3f4a17df
371f259c-3f4a17df
d990c4ac-3f4a17df
5c7c8339-3f4a17df
a18444ea-a18444ea
ef4764d7-c9f4d4ef
9909b8ac-3f4a17df
e52d4c86-3f4a17df
1fce7d57-3f4a17df
7760b5b2-3f4a17df
e8c68789-49a20295
caa76e48-caa76e48
b0f15b33-b0f15b33
94f1fa38-66264bee
2b0207ee-2b0207ee
ad4acdda-3f4a17df
90860314-3f4a17df
931c5f72-3f4a17df
7ec047c2-3f4a17df
ade3efeb-e1cc0f14
f48c01d3-6eb2bd2b
b1ceb06f-3f4a17df
db59f83a-3f4a17df
57675af7-3f4a17df
ec21b181-3f4a17df
160d8d8d-9d12ca0c
17aec314-3f4a17df
1d0518a-3f4a17df
7f6bf080-3f4a17df
6becb1e-a6ea97a2
f6e27768-f695fd79
3b96a1d-3f4a17df
dba92675-f23d1dea
595f5eb0-f23d1dea
bef5c006-3f4a17df
c98abd03-3bf4b625
a8cfdb40-3f4a17df
17b84626-3f4a17df
8d7344de-3f4a17df
2856aa31-3f4a17df
e11c0dd9-3586c521
```
</details>


<details><summary>Chromium 113.0.5618.0 (Developer Build) (arm64)</summary>
2510663e-73703436
df319cb2-27e0e1e8
8963b549-3f4a17df
3e7d7783-a4655118
e5726113-3f4a17df
3e8fe123-3f4a17df
15e85316-3f4a17df
aa21b4f1-3f4a17df
d6284ba0-3be74c4d
9d165456-3f4a17df
13e2821-c0236c9e
eddd0d82-3f4a17df
13427e22-3f4a17df
f2855e3d-de2b6078
8c153587-3f4a17df
870c1db8-3f4a17df
1d167f26-3f4a17df
42f0e0ea-75d6947c
9785b8cf-41973d1d
87f33ad6-3f4a17df
afb5d7b8-3f4a17df
4d936449-fd549ada
1bb6a450-3f4a17df
250dda8b-3f4a17df
f14c322f-f9a43703
5be458ba-78266b16
9f476f76-3f4a17df
78634d5f-2ba5dc1c
4749874c-455e925b
cd59be9a-6e5c8f84
7c2504d0-3f4a17df
9a24c3c2-7656653c
251b2579-3f4a17df
1acce950-3f4a17df
206d80d-3f4a17df
a2fd384c-cc71bb94
e521d2ef-3f4a17df
55ba4cfa-3f4a17df
f654ad46-c94f66c2
f8b7087b-3d47f4f4
5c4d440e-58c8ac88
fd051c38-3f4a17df
cbc04857-3f4a17df
65570806-377be55a
42f1f10d-98837767
363012bc-3f4a17df
92d2eb18-fa10226e
18324944-3f4a17df
520b2a89-88bf9f37
1584cf60-3f4a17df
81b1a2c3-3f4a17df
8c82550d-3f4a17df
dfa02032-7ae17237
8ddeb5cd-3f4a17df
565dffc5-565dffc5
d92d97b4-e913bec6
4852ec7f-3f4a17df
263848e4-3f4a17df
3042ad4b-ad2fa222
3fd33f16-27b09c4c
c297985a-3f4a17df
ae850472-3f4a17df
cb50325a-52bfb160
19d422be-c9846617
72fb77ee-3f4a17df
e0e63e5f-decb98bf
911e33b9-3f4a17df
9982045c-3f4a17df
95bc6922-3f4a17df
cf64b238-3f4a17df
36d5ee52-3f4a17df
7a842d4a-b3a17800
26db9e79-f430c221
17cd3426-3f4a17df
2f7e7ede-3f4a17df
910af27f-3f4a17df
8bccc03b-3f4a17df
e45cd2f5-3f4a17df
83b371a5-3f4a17df
5349039a-3f4a17df
d664a1aa-3f4a17df
625646ce-3f4a17df
53c131d9-e3ef03f6
a7780b9-3f4a17df
fcad39d1-3f4a17df
a79ba57a-f23d1dea
9e5c75f1-30e1b12b
c4e32a07-3a3251a9
255dfea8-cf12f279
ed2101b4-3f4a17df
3095b8fe-3f4a17df
75ffb03a-3f4a17df
1527f29f-3f4a17df
b3c9749a-3f4a17df
25fa4483-b9406639
ca5a2953-ff983c32
da493d3c-3f4a17df
f3ed486d-3f4a17df
5c783e42-8f8fa88f
b3c54bb3-a058b5d3
bde7927d-bab20a64
7989c213-3f4a17df
4ff8f5b5-caf7a452
de2c80ca-da5d16c4
ca12356a-3f4a17df
741e95d4-3f4a17df
7b9d3e0c-7f734c6e
d3566fbd-c6f74b94
a779bb20-3f4a17df
ea23a088-41c37e8a
b16eab85-95ab9b79
e898a92c-3f4a17df
5f36436a-f799c15e
7fb629a1-60fdb59
2da2abac-b7f59038
47be28a0-3f4a17df
186d6e2c-36c0e608
4ea303a6-3f4a17df
4949c1d8-3f4a17df
e3fd1192-3f4a17df
de95f00f-3f4a17df
baee3c29-3f4a17df
9fe21c85-3f4a17df
68f0d29d-3f4a17df
15d3083e-5ce60213
f08c130d-3f4a17df
d5a65352-3f4a17df
c9e4cf65-f750a0e4
f112d133-3f4a17df
9a564e2a-3f4a17df
b5663e5f-3f4a17df
31af02a2-3f4a17df
371f259c-3f4a17df
9635fb98-3f4a17df
d990c4ac-3f4a17df
5c7c8339-3f4a17df
f0682056-324bfab2
ef4764d7-c9f4d4ef
9909b8ac-3f4a17df
e52d4c86-3f4a17df
5851a53a-8897814c
1fce7d57-3f4a17df
7760b5b2-3f4a17df
e8c68789-49a20295
caa76e48-caa76e48
b0f15b33-b0f15b33
94f1fa38-66264bee
ad4acdda-3f4a17df
90860314-3f4a17df
931c5f72-3f4a17df
7ec047c2-3f4a17df
ade3efeb-e1cc0f14
b1ceb06f-3f4a17df
db59f83a-3f4a17df
57675af7-3f4a17df
ec21b181-3f4a17df
160d8d8d-9d12ca0c
bbaef9b4-3f4a17df
4810967e-3f4a17df
1d0518a-3f4a17df
7f6bf080-3f4a17df
6becb1e-a6ea97a2
f6e27768-f695fd79
3b96a1d-3f4a17df
595f5eb0-f23d1dea
bef5c006-3f4a17df
a8cfdb40-3f4a17df
17b84626-3f4a17df
8d7344de-3f4a17df
e11c0dd9-3586c521
</details>

**Comment by mfreed7** [comment]
Yep! That's it. 263848e4-3f4a17df (present in all) is Popover-enabled. I guess the Chromium builds you are using are on the Dev channel and get Finch updates. And those get the experiment I ran for compatibility of `popover` as an attribute.

Sorry for the trouble. But I'm very confident that production stable builds of Chrome did not have the attribute enabled prior to 114. You can test by explicitly passing `--disable-blink-features=HTMLPopoverAttribute` on the command line, which will override any Finch config.
